### PR TITLE
Uz/guide/getting started

### PR DESCRIPTION
**Pull request by Dostonlv**

### CODE REVIEW

File: site/docs/.vuepress/config.ts [hunk 1]
```diff
@@ -700,7 +1032,50 @@ export default defineUserConfig<DefaultThemeOptions>({
...
+                noResultsScreen: {
+                  noResultsText: "So'rov bo'yicha natija mavjud emas",
+                  suggestedQueryText: "Quyidagini qidirishga urinib ko'ring",
+                  openIssueText:
+                    "Bu so'rovdan natija chiqishiga ishonchingiz komilmi?",
+                  openIssueLinkText: "Bizga xabar bering",
+                },
+              },
+            },
+          },
```
Comment (KnorpelSenf) [inline site/docs/.vuepress/config.ts:1077]: @WingLim @linbuxiao @taotie111 @JiquanWang99 could we also have these translations for Chinese (in a second PR)? The concern the Algolia search interface. We missed this in the early days when we were translating the config.

Comment (KnorpelSenf) [reply site/docs/.vuepress/config.ts:1077]: cc @kidonng

Comment (KnorpelSenf) [reply site/docs/.vuepress/config.ts:1077]: @IamItachiUchiha @mcpeblocker @sad1go0 could you leave a link here which points to the English originals of this configuration that you used when translating this? Such as the Algolia API reference or something.

Comment (mcpeblocker) [reply site/docs/.vuepress/config.ts:1077]: Here:
https://docsearch.algolia.com/docs/api/#translations

**Comment by mcpeblocker** [comment]
Sorry for that PR :(

**Pull request closed without merging**